CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 All notable changes to this project will be documented in this file.
 
+## v0.4.0
+
+_Release: 2019-05-20_
+
+##### New Features
+
+- `Card` and `Divider` components
```

Comment:
divider is not listed in docs (cause it was developed ad hoc), should we annouced this in changelog?

Reply:
it doesn't have a dedicated page but AFAIR it's used in `Card` page. IMHO we should include it here. What ya think?

Reply:
i have mixed feelings, could stay but we should provide a dedicated page in docs in the nearest future.

Reply:
Ok, I will add it tomorrow :)